src/main/java/io/anserini/search/SearchArgs.java

```diff
@@ -186,4 +186,11 @@
 
   @Option(name = "-model", metaVar = "[file]", required = false, usage = "ranklib model file")
   public String model = "";
+
+  @Option(name = "-qlmj", usage = "use Jelinek-Mercer query likelihood scoring model")
+  public boolean qlmj = false;
+
+  @Option(name = "-lambda", metaVar = "[value]", usage = "Jelinek Mercer smoothing parameter")
```

Comment:
Can we rename it to `qlmj.lambda`?
I know for BM25 and QL the parameters were `b` and `mu` and we should have changed them too. 
But those two were there from the very beginning of Anserini and breaking them will cause problem of regression tests.
If you look at other ranking functions like `pl2` you will see its parameter as `-pl2.c`.

Also, since `qlmj` is a _base_ ranking model could you please move these two options right under `ql`? 

Thanks

Reply:
sure, will do

Reply:
Yes, and technically `-ql` should be `-qld` for (Query Likelihood Dirichlet). Such a change will likely break lots of regression testing scripts... :(

But perhaps worth filing an issue?

Reply:
maybe `-qld` can be added, which would work exactly like `-ql` for backward compatibility.

Reply:
This is a good idea.